src/styles/blocks/button.scss

```diff
@@ -0,0 +1,36 @@
+.button {
+  cursor: pointer;
+  transition: 0.5s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.